components/admin_console/custom_enable_disable_guest_accounts_setting.tsx

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+
+import FormattedMarkdownMessage from 'components/formatted_markdown_message';
+
+import ConfirmModal from 'components/confirm_modal.jsx';
+
+import BooleanSetting from './boolean_setting';
+
+type Props = {
+    id: string;
+    value: boolean | string;
```

Comment:
Not strictly related to this PR but why do we have to handle string booleans?

Reply:
Done, not needed to have both.